Source/ACE.Server/Factories/LootGenerationFactory_Test.cs

```diff
@@ -249,8 +249,10 @@ public static LootStats LootStats(WorldObject wo, LootStats ls, bool logstats)
                                 petLevel = 100;
                             else if (petDevice.UseRequiresSkillLevel == 370)
                                 petLevel = 80;
-                            else if (petDevice.UseRequiresSkillLevel == 310)
+                            else if (petDevice.UseRequiresSkillLevel == 310 || petDevice.UseRequiresSkillLevel == 320)
                                 petLevel = 50;
+                            if (petLevel == 0)
+                                Console.WriteLine("Skill= " + petDevice.UseRequiresSkillLevel);
```

Comment:
debug code?

Reply:
Some PetDevice weenies do not have the correct wield reqs, thats why its there

Reply:
shouldn't that be using log.Debug then, and printing out a full error message?

if Skill=# gets output to the console, it will be unclear what that indicates